src/App.xaml

```diff
@@ -15,7 +15,7 @@
                 <ResourceDictionary Source="/Styles/Thickness.xaml" />
                 <ResourceDictionary Source="/Styles/WindowTitleBar_ThemeResources.xaml" />
 
-                <!-- Resources from other projects -->
+                <!--  Resources from other projects  -->

```

Comment:
nit: I'm not sure if we're going to go with the 2 space buffers as standard, so I'd rather not change this now.

Reply:
This is the work of the xaml formatter extension.  It adds the second space on a save.  I'll get this fixed.